jcstress-samples/src/main/java/org/openjdk/jcstress/samples/high/race/condition/RaceCondition_01_ReadModifyWriteSequence.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.jcstress.samples.high.race.condition;
+
+import org.openjdk.jcstress.annotations.Actor;
+import org.openjdk.jcstress.annotations.JCStressTest;
+import org.openjdk.jcstress.annotations.Outcome;
+import org.openjdk.jcstress.annotations.State;
+import org.openjdk.jcstress.infra.results.III_Result;
+
+import static org.openjdk.jcstress.annotations.Expect.*;
+
+/*
+    How to run this test:
+        $ java -jar jcstress-samples/target/jcstress.jar -t RaceCondition_01_ReadModifyWriteSequence
+ */
+
+/**
+ * This sample demonstrates you how a read-modify-write sequence can lead to surprising results.
+ */
+@JCStressTest
+@Outcome(id = {"150, 100, 150"}, expect = ACCEPTABLE, desc = "Actor1 considered actor2's result and wrote his right result")
+@Outcome(id = {"250, 150, 150"}, expect = ACCEPTABLE, desc = "Actor2 considered actor1's result and wrote his right result")
+@Outcome(id = {"250, 100, 250", "250, 150, 250"}, expect = ACCEPTABLE_INTERESTING, desc = "Actor1 ignored actor2's result and wrote his wrong result")
+@Outcome(id = {"250, 100, 100", "150, 100, 100"}, expect = ACCEPTABLE_INTERESTING, desc = "Actor2 ignored actor1's result and wrote his wrong result")
```

Comment:
Please use gender-neutral pronouns or avoid using pronouns altogether (e.g. see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/inclusive_code.md )

Reply:
Yeah, that makes sense. I replaced his by its. Is that better?

Reply:
I would rather write the description like in pseudo-code.
For instance: `"actor2: value=50, then actor2: value+=50"`

Remember that the message would appear in the test output, and it would be great to be able to understand the meaning of the outcomes **without** analyzing the source code.

Reply:
Indeed. I like your idea. I changed and committed RaceCondition_01_ReadModifyWriteSequence.
What do you think about the new version?
Please keep in mind that the space for the decription is limited and cutting of the last part of the description would be bad.

Reply:
Can we resolve that now, @vlsi?